IMDA-BTG/aiverify
# add radar chart and treemap

## Description

Add new charts:
- Treemap
- Radar Chart

## Motivation and Context

Additional charts for widgets.

## Type of Change













## How to Test

Create test plugin and new widgets to test the new charts. Examples provided in the Charts documentation: https://github.com/IMDA-BTG/aiverify/tree/main/ai-verify-shared-library/packages/charts.

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]
